### ISSUE TITLE
[6.0.0] Insane number of events being emitted

### ISSUE THREAD

**Issue by tetshi** [ankurk91/vue-flatpickr-component#53]
This thing is emitting events like crazy. @on-day-create emits 100's of times on page load (multiply this by each instance of Flatpickr on the page), as well as @on-parse-config and @on-ready. This is horrible for performance and it kills the Vue debugger in Chrome.

**Comment by ankurk91** [comment]
@tetshi 
What about allowing you to choose which events to emit?

There will be a prop called `:events`, where you can pass array of events.
```html
<flat-pickr v-model="date" :config="options" :events="['onChange']">
```
So now only `on-change` event will be emitted.

**Comment by ankurk91** [comment]
ping @tetshi

**Comment by ankurk91** [comment]
Okay, a new release is coming soon.

**Comment by ankurk91** [comment]
fixed in version v6.1.0
